quickstart/k8s-the-prod-way/identity.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Bootstrapping Identity
+---
+
+Begin by cloning the [Kubernetes the Prod Way repository][ktpw]. The `identity` stack is implemented
+for each of [AWS][aws], [Azure][azure], and [GCP][gcp].
+
+This stack is the root from which we will bootstrap the rest of our infrastructure. In this
+tutorial, we will:
+
+* Provision service accounts for CI/CD
+* Provision roles for users and teams
+
+**IMPORTANT CAVEAT:** The AWS IAM role and Azure AD account APIs are not do not yet fully exposed
+through the Pulumi SDK. This stack will provision them, but you will have to manually grant them
+permissions.
+
+## Prerequisites: Bootstrapping Identity
+
```

Comment:
What about on-prem?

Reply:
This tutorial was purposefully scoped to public clouds.

Reply:
Why? Many of our users/customers will be doing this on-prem. It at least warrants a mention (coming soon, out of scope, ...)